plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestOracleCaseInsensitiveMapping.java

```diff
@@ -54,123 +55,47 @@ protected QueryRunner createQueryRunner()
-                    ImmutableSet.of("lower_case_name", "mixed_case_name", "upper_case_name"));
-
-            // Note: until https://github.com/prestodb/presto/issues/2863 is resolved, this is *the* way to access the tables.
-
-            assertQuery("SELECT lower_case_name FROM someschema.nonlowercasetable", "VALUES 'a'");
-            assertQuery("SELECT mixed_case_name FROM someschema.nonlowercasetable", "VALUES 'b'");
-            assertQuery("SELECT upper_case_name FROM someschema.nonlowercasetable", "VALUES 'c'");
-            assertQuery("SELECT upper_case_name FROM SomeSchema.NonLowerCaseTable", "VALUES 'c'");
-            assertQuery("SELECT upper_case_name FROM \"SomeSchema\".\"NonLowerCaseTable\"", "VALUES 'c'");
-
-            assertUpdate("INSERT INTO someschema.nonlowercasetable (lower_case_name) VALUES ('l')", 1);
-            assertUpdate("INSERT INTO someschema.nonlowercasetable (mixed_case_name) VALUES ('m')", 1);
-            assertUpdate("INSERT INTO someschema.nonlowercasetable (upper_case_name) VALUES ('u')", 1);
-            assertQuery(
-                    "SELECT * FROM someschema.nonlowercasetable",
-                    "VALUES ('a', 'b', 'c')," +
-                            "('l', NULL, NULL)," +
-                            "(NULL, 'm', NULL)," +
-                            "(NULL, NULL, 'u')");
-        }
-    }
-
-    @Test
-    public void testSchemaNameClash()
-            throws Exception
+    @Override
+    protected SqlExecutor onRemoteDatabase()
     {
-        String[] nameVariants = {"\"casesensitivename\"", "\"CaseSensitiveName\"", "\"CASESENSITIVENAME\""};
-        assertThat(Stream.of(nameVariants)
-                .map(name -> name.replace("\"", "").toLowerCase(ENGLISH))
-                .collect(toImmutableSet()))
-                .hasSize(1);
-
-        for (int i = 0; i < nameVariants.length; i++) {
-            for (int j = i + 1; j < nameVariants.length; j++) {
-                String schemaName = nameVariants[i];
-                String otherSchemaName = nameVariants[j];
-                try (AutoCloseable ignore1 = withSchema(schemaName);
-                        AutoCloseable ignore2 = withSchema(otherSchemaName);
-                        AutoCloseable ignore3 = withTable(schemaName + ".some_table_name", "(c varchar(5))")) {
-                    assertThat(computeActual("SHOW SCHEMAS").getOnlyColumn().filter("casesensitivename"::equals)).hasSize(1); // TODO change io.trino.plugin.jdbc.JdbcClient.getSchemaNames to return a List
-                    assertQueryFails("SHOW TABLES FROM casesensitivename", "Failed to find remote schema name: Ambiguous name: casesensitivename");
-                    assertQueryFails("SELECT * FROM casesensitivename.some_table_name", "Failed to find remote schema name: Ambiguous name: casesensitivename");
-                }
-            }
-        }
+        return oracleServer::execute;
     }
 
-    @Test
+    @Test @Override
```

Comment:
Can you please add a comment why method is overriden?

Please put each annotation in separate line